tests/python_functional/src/driver_io/file/file_io.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, logger_factory, file_path):
         self.__readable_file = None
         self.__writeable_file = None
 
-    def read(self, position=None):
+    def read(self, position=None, counter=None):
```

Comment:
counter would be used for position tracking. In that sense, it would worth reusing the position.

Reply:
Valid, +1

Reply:
Valid, +1

Reply:
done